pyproject.toml

```diff
@@ -263,9 +273,7 @@ filterwarnings = [
     "once:The behavior of DataFrame concatenation with empty or all-NA entries is deprecated.:FutureWarning",
 ]
 
-markers = [
-    "slow: marks tests as slow (deselect with '-m \"not slow\"')",
-]
+markers = ["slow: marks tests as slow (deselect with '-m \"not slow\"')"]
```

Comment:
Not sure why this is all in red. Just a change in line-wrapping from my autoformatter.

Reply:
Not sure either but it's very scary.